dashboard/src/app/[tenantId]/userTaskDef/[...props]/components/UserTaskDef.tsx

```diff
@@ -178,6 +179,17 @@ export const UserTaskDef: FC<Props> = ({ spec }) => {
                               ? utcToLocalDateTime(userTaskRun.scheduledTime)
                               : NOT_APPLICABLE_LABEL}
                           </TableCell>
+                          <TableCell>
+                            <Button
+                              onClick={() =>
+                                router.push(
+                                  `/${tenantId}/userTaskDef/audit/${userTaskRun.id?.wfRunId?.id}/${userTaskRun.id?.userTaskGuid}`
+                                )
+                              }
+                            >
+                              View Audit Log
+                            </Button>
```

Comment:
Same with this, please use `LinkWithTenant` and if you want the default styling of `Button` then you can use `asChild` prop and put the `Button` as a child of `LinkWithTenant`